content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md

```diff
@@ -0,0 +1,55 @@
+# How to Add Dependencies or Outputs
```

Comment:
add to something?

Reply:
Unfortunately the title is too long so I felt like we needed to compromise. See nav vs. title here:

![image](https://user-images.githubusercontent.com/1477535/100289387-5bd38a00-2f3e-11eb-96a6-419b5df9dbb8.png)

I've added a SEO `title` now that is more correct though. Should help

Reply:
p.s. on this, I think it's OK since "Dependencies or Outputs" can only apply to stages anyway.